<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaEditar</class>
 <widget class="QWidget" name="TelaEditar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lFoto">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>130</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;h3&gt;Foto do aluno:&lt;/h3&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterarFoto">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar foto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterarDados">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Alterar Dados</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVoltar">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar para o menu</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>421</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;h3&gt;RM do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leRmAluno">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: red;
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>&lt;h3&gt;Nome completo do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNomeAluno"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;h3&gt;Data de nascimento do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="deDataNascimento">
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;h3&gt;Sexo do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="rbMasculino">
        <property name="text">
         <string>Masculino</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbFeminino">
        <property name="text">
         <string>Feminino</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>&lt;h3&gt;RG do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leRGAluno">
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: red;
}</string>
      </property>
      <property name="inputMask">
       <string>99.999.999-9</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>&lt;h3&gt;CPF do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leCPFAluno">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	color: red;
}</string>
      </property>
      <property name="inputMask">
       <string>999.999.999-99</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;h3&gt;Nome do responsavel do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leNomeResponsavel"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>&lt;h3&gt;Telefone do responsavel do aluno:&lt;/h3&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leTelefoneResponsavel">
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="inputMask">
       <string>(99) 99999-9999</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
